interface/billing/edi_history_main.php

```diff
@@ -100,6 +101,14 @@
  */
 
 if (strtolower($_SERVER['REQUEST_METHOD']) == 'post') {
+    if (!empty($_POST)) {
+        if (!isset($_POST['token'])) {
+            error_log('WARNING: A POST request detected with no csrf token found');
+            die('Authentication failed.');
+        } else if (!(CsrfToken::verifyCsrfToken($_POST['token'])) {
```

Comment:
You're missing a closing parenthesis here

Reply:
thanks @maggienegm. I am guessing you fixed in here PR #1634?? have not checked yet. If you did then I can just close this.

Reply:
Yes, fixed the closing parenthesis issues there